core/config.py

```
@@ -0,0 +1,205 @@
+        try:
+            # 加载主配置
+            self.main_config = self._load_toml_file(self.main_config_path)
+            logger.info(f"✓ 加载主配置: {self.main_config_path}")
+            
+            # 加载模块配置
+            modules = self.main_config.get("modules", {})
+            for module_name, config_path in modules.items():
+                try:
+                    self.module_configs[module_name] = self._load_toml_file(config_path)
+                    logger.info(f"✓ 加载模块配置: {module_name} -> {config_path}")
+                except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)